syft/frameworks/torch/mpc/__init__.py

```diff
@@ -29,6 +31,7 @@ def decorator(f):
         name = f.__qualname__
         protocol_store[(name, protocol_name)] = f
 
+        @wraps(f)
```

Comment:
Why is this required?

Reply:
For using 2 decorators on same function.